public/pages/utils/__tests__/constants.ts

```diff
@@ -134,3 +134,171 @@ export const selectedDetectors = [
     lastUpdateTime: 1651818220194,
   },
 ] as DetectorListItem[];
+
+export const anomalyResultSummary = {
```

Comment:
nit: I prefer all constant vars to be in ALL_CAPS so its easy to differentiate and makes the tests where these constants are used a bit more readable